london.html

```diff
@@ -60,21 +45,11 @@
         <!-- Side nav for mobile - menu links MUST BE REPEATED HERE,
         hidden in non-mobile settings-->
         <ul class="mobile-nav">
-          <li>
-            <a href="index.html">Home</a>
-          </li>
-          <li>
-            <a href="chapters.html">Chapters</a>
-          </li>
-          <li>
-            <a href="events.html">Upcoming Events</a>
-          </li>
-          <li>
-            <a href="resources.html">Resources</a>
-          </li>
-          <li>
-            <a href="./about.html">About Us</a>
-          </li>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="chapters.html">Chapters</a></li>
+          <li><a href="events.html">Upcoming Events</a></li>
+          <li><a href="resources.html">Resources</a></li>
+          <li><a href="#footer">Contact Us</a></li>
```

Comment:
Same as above

Reply:
Also I've realised this one says "Upcoming Events" when the other navs just say "Events"